src/Administration/Resources/app/administration/src/global.types.ts

```diff
@@ -195,7 +197,7 @@ declare global {
     interface CustomShopwareProperties {}
```

Comment:
The interface is already there and ShopwareClass extends it - I added this a year ago, why is it a problem now?

Reply:
*And back then it was working fine :thinking:

Reply:
I tried it out but for me it didn't worked when accessing it directly on the Shopware object.

Reply:
okay, still `CustomShopwareProperties` is now duplicated defined :smile:

Reply:
I updated it. Now it is defined only once. But I needed it to move upwards